internal/config/settings.go

```diff
@@ -24,4 +24,5 @@ type Settings struct {
 	DrivlyVINAPIURL                string         `yaml:"DRIVLY_VIN_API_URL"`
 	DrivlyOfferAPIURL              string         `yaml:"DRIVLY_OFFER_API_URL"`
 	Redis                          redis.Settings `yaml:"REDIS"`
+	NHTSARecallsFileURL            string         `yaml:"NHTSA_RECALLS_FILE_URL"`
```

Comment:
add a setting for the kubernetes to pull for this in `/charts/device-definitions-api/values.yaml` under the `env:` section. Since there isn't a different value for production you only need to set it there.